pyqtgraph/graphicsItems/ScatterPlotItem.py

```diff
@@ -629,6 +722,25 @@ def updateSpots(self, dataSet=None):
         if invalidate:
             self.invalidate()
 
+    def _style(self, opts, data=None, idx=None, scale=None):
+        if data is None:
+            data = self.data
+
+        if idx is None:
+            idx = np.s_[:]
+
+        for opt in opts:
+            col = data[opt][idx]
+            if col.base is not None:
+                col = col.copy()
+            col[np.equal(col, _DEFAULT_STYLE[opt])] = self.opts[opt]
+
+            if opt == 'size' and scale is not None:
+                col *= scale
+
+            yield col
+
+    # deprecated
```

Comment:
If actually deprecated; we should emit a deprecation warning.

Reply:
I'm going to take that comment out. This method is replaced internally by _style, but there isn't a public replacement.